src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,139 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_COEFFICIENT = 2;
+    private Node<K, V>[] myHashMap;
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        myHashMap = new Node[INITIAL_CAPACITY];
+        threshold = (int) (myHashMap.length * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resize();
+        }
+        Node<K, V> newNode = new Node<>(key, value, null);
+        putNode(newNode);
+        size++;
     }
 
     @Override
     public V getValue(K key) {
-        return null;
+        V value = null;
+        for (Node<K, V> node : myHashMap) {
+            if (node == null) {
+                continue;
+            }
+            while (node != null) {
+                if (Objects.equals(key, node.key)) {
+                    value = node.value;
+                }
+                node = node.next;
+            }
```

Comment:
No sense if-statement, if the node is null, it won't enter in while-loop, and the main foreach-loop will continue iteration